src/backend/distributed/commands/foreign_constraint.c

```diff
@@ -1010,3 +966,30 @@ GetReferencedTableId(Oid foreignKeyId)
 
 	return referencedTableId;
 }
+
+
+/*
+ * IsTableTypeIncluded returns true if type of the table with relationId (distributed,
+ * reference, Citus local or Postgres local) is included in the flags, false if not
+ */
+static bool
+IsTableTypeIncluded(Oid relationId, int flags)
+{
+	if (!IsCitusTable(relationId))
+	{
+		return flags & INCLUDE_LOCAL_TABLES;
```

Comment:
minor: it's nicer to say (flags & INCLUDE_LOCAL_TABLES) != 0, to ensure that bool is either 0 (false) or 1 (true), rather than some arbitrary number.